### ISSUE TITLE
Fails to build on recent nightly (2018-07-17)

### ISSUE THREAD

**Issue by pitdicker** [koute/stdweb#252]
```
   Compiling stdweb-internal-macros v0.1.0
error[E0432]: unresolved import `stdweb_internal_macros::js_export`
   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/stdweb-0.4.7/src/lib.rs:136:9
    |
136 | pub use stdweb_internal_macros::js_export;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `js_export` in the root
error: aborting due to previous error
For more information about this error, try `rustc --explain E0432`.
error: Could not compile `stdweb`.
error: build failed
```

https://travis-ci.org/rust-lang-nursery/rand/jobs/404778291

Maybe related to https://github.com/rust-lang/rust/pull/52383?

**Comment by MaxGraey** [comment]
Same problem

**Comment by pitdicker** [comment]
You are always very quick, thank you!

**Comment by koute** [comment]
Published as `0.4.8`!
